CHANGELOG.asciidoc

```diff
@@ -141,6 +141,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Add autodiscover for kubernetes. {pull}6055[6055]
 - Add the abilility for the add_docker_metadata process to enrich based on process ID. {pull}6100[6100]
 - The `add_docker_metadata` and `add_kubernetes_metadata` processors are now GA, instead of Beta. {pull}6105[6105]
+- Use last resource version when watching for kubernetes events, to avoid duplicates. {pull}6230[6230]
```

Comment:
Is this changelog even neded as k8s watcher was not in a previous version I think?